routes/users.js

```diff
@@ -1,28 +1,28 @@
-const express = require('express');
-const passport = require('passport');
-const token = require('../utils/token'); 
-require('../config/facebook');
+const express = require("express");
+const authService = require("../service/Auth");
 
 // ToDo:// Store AuthToken in DB
 // Mark: Callback to generate access token
-function generateUserToken(req, res) {
-	console.log(`generating token for user ID ${req.user.id}`);
-	const accessToken = token.generateAccessToken(req.user.id);
-	console.log(`access Token: ${accessToken}`);
-	res.json(accessToken);
-}
 
 const router = express.Router();
 
-router.get('/auth/facebook', passport.authenticate('facebook', { scope: ['email'], session: false }));
+router.get("/auth", (req, res, next) => {
+  new authService(req.query.name).getAuthorize()(req, res, next);
+});
 
 // Todo: Apply redirect URL
-router.get('/auth/facebook/callback',
-	passport.authenticate('facebook', { session: false, failureRedirect: '/fail' }),
-	generateUserToken);
+router.get(
+  "/auth/callback",
```

Comment:
We are using 4 tab indent don't use spaces, Please enable ESlint.

Reply:
this problem is due to react plugins that i am using which have norms according to react on autosave they changes will see to it